client/components/thank-you/index.tsx

```diff
@@ -40,6 +39,11 @@ const ThankYouHeader = styled.div`
     justify-content: center;
 }`;
 
+const ThankYouHeaderTitle = styled.h1`
+	font-size: ${ ( { subtitle = false } ) => ( subtitle ? '1.5em' : '2em' ) };
+	margin-bottom: 8px;
+`;
+
 const ThankYouSectionContainer = styled.div`
     margin-bottom: 35px;
 }`;
```

Comment:
I think this felt through the cracks - there shouldn't be any closing brace here (same thing line 40).